Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM node:20.5-alpine as node
+FROM docker.io/node:20.5-alpine as node
 COPY . /shlink-web-client
 ARG VERSION="latest"
 ENV VERSION ${VERSION}
 RUN cd /shlink-web-client && npm ci && npm run build
 
-FROM nginx:1.25-alpine
+FROM docker.io/nginxinc/nginx-unprivileged:1.25-alpine
```

Comment:
I'm in favor of distributing a non-privileged docker image, but is this an official image? Can you provide more context on why did you choose this particular image?

Reply:
Sure, this is the official nginx unprivileged container. You can follow the trail from [1] to [2]. 

On 1 it states:
> Alternatively, check out the official Docker NGINX unprivileged image.

The GitHub source[3] for those images available too.

[1] https://hub.docker.com/_/nginx
[2] https://hub.docker.com/r/nginxinc/nginx-unprivileged
[3] https://github.com/nginxinc/docker-nginx-unprivileged

Reply:
Perfect! Thanks for the clarification. Let's just add it as `nginxinc/nginx-unprivileged:1.25-alpine` (no registry name) for consistency.